ports/atmel-samd/boards/arduino_mkrzero/mpconfigboard.mk

```diff
@@ -1,6 +1,6 @@
 LD_FILE = boards/samd21x18-bootloader.ld
 USB_VID = 0x239A
-USB_PID = 0x8035
+USB_PID = 0x8050
```

Comment:
why does the arduino use an adafruit vid?

Reply:
We’re replacing the regular bootloader and don’t have multiple pids available from Arduino. This was assigned a long time ago but had errors.

Reply:
kk